versions.yaml

```diff
@@ -115,6 +110,12 @@ assets:
       url: "https://github.com/kata-containers/qemu"
       tag: "ad4c7f529a279685da84297773b4ec8080153c2d-plus-TDX-v1.3"
 
+    qemu-snp-experimental:
+      description: "QEMU with experimental SNP support (no UPM)"
+      url: "https://github.com/AMDESE/qemu"
+      branch: "snp-v3"
+      tag: "3b6a2b6b7466f6dea53243900b7516c3f29027b7"
```

Comment:
And yep, it does need to be an experimental one.

Reply:
I don't think `branch` is actually used to clone/checkout the tree, it does? If not, it can be removed.

Reply:
Yeah not necessary.